src/lib/components/DeckGLMap/layers/wells/wellsLayer.ts

```diff
@@ -14,18 +16,104 @@ export interface WellsLayerProps<D> extends CompositeLayerProps<D> {
     outline: boolean;
     selectedFeature: Feature;
     selectionEnabled: boolean;
+    logData: string;
```

Comment:
This could come either as an url (so string), or directly as the object data. You can move the `LogCurveDataType` above `WellsLayerProps<D>` and have the `logData: string | LogCurveDataType`.